src/Ledger/Epoch.lagda

```diff
@@ -156,9 +146,83 @@ toRwdAddr x = record { net = NetworkId ; stake = x }
 getStakeCred : TxOut → Maybe Credential
 getStakeCred (a , _ , _ , _) = stakeCred a
 
-open RwdAddr using (stake)
 open GovActionState using (returnAddr)
 \end{code}
+\begin{NoConway}
+\Cref{fig:functions:createRUpd} defines
+the function \AgdaFunction{createRUpd}
+which creates a \AgdaRecord{RewardUpdate},
+i.e.\ the net flow of Ada due to paying out rewards
+after an epoch.
+Relevant quantities are:
+\begin{itemize}
+  \item \AgdaArgument{prevPp}: Previous protocol parameters,
+    which correspond to the parameters during the epoch for which we are creating rewards.
+  \item \AgdaFunction{Δr₁}: Ada taken out of the reserves for paying rewards,
+    as determined by the \AgdaField{monetaryExpansion} protocol parameter.
+  \item \AgdaFunction{rewardPot}: Total amount of coin available for rewards this epoch, as
+described in section 6.4 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{feeSS}: The fee pot, containing the transaction fees from the epoch.
+    We use the fee pot from the snapshot taken at the epoch boundary.
+    Note that fees are not explicitly removed from any account:
+    the fees come from transactions paying them and are accounted for whenever transactions are processed.
+  \item \AgdaFunction{Δt₁}: The proportion of the reward pot that will move to the treasury,
+    as determined by the \AgdaField{treasuryCut} protocol parameter.
+    The remaining pot is called the \AgdaFunction{R},
+    just as in section 6.5 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{pstakego}:
+    Stake distribution used for calculating the rewards.
+    This is the oldest stake distribution snapshot, labeled ``go''.
+  \item \AgdaFunction{rs}: The calculated rewards.
+    As given by \AgdaFunction{maxPool}, each pool can receive a maximal amount,
+    determined by its performance.
```

Comment:
This phrase is a bit difficult to understand the way its written. Also, it talks about this quantity determined by `maxPool`, how are these connected?

Reply:
I have removed the mention of `maxPool` in favor of simply delegating the discussion to `reward`.